app/controllers/posts_controller.rb

```diff
@@ -1,4 +1,5 @@
 class PostsController < ApplicationController
+  load_and_authorize_resource
   def index
     @user = User.includes(:posts).find(params[:user_id])
     @posts = @user.posts.includes(:user)
```

Comment:
- [ ]  These two lines are throwing errors in your code because you used the wrong syntaxes. kindly use this code for example to get the user and the posts 👇 

```rb
## make sure you update your database by changing `users_id` to `user_id`
    @user = User.find(params[:user_id])
    @posts =  @user.posts.includes(:comments)
```

Reply:
sure let me check